src/source.py

```diff
@@ -16,4 +16,82 @@ def d2YdtdE_0_func_doublegauss(t, E, N, mu_t, sig_t, mu_E, sig_E):
 
 def dYdt_0_func_gauss(t, E, N, mu_t, sig_t):
     dYdt_0 = N/np.sqrt(2*np.pi)/sig_t * np.exp(-0.5 * ((t-mu_t)/sig_t)**2)
-    return dYdt_0
\ No newline at end of file
+    return dYdt_0
+
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Feb 20 09:27:10 2023
+
+@author: blr
+"""
+import numpy as np
+import scipy
+
+##############################################################################
+########################### MODEL FUNCTIONS ##################################
+##############################################################################
+
+def d2YdtdE_0_func_doublegauss(t, E, N, mu_t, sig_t, mu_E, sig_E):
+    d2YdtdE_0 = N/2/np.pi/sig_t/sig_E * np.exp(-0.5 * (((t-mu_t)/sig_t)**2 + ((E-mu_E)/sig_E)**2))
+    return d2YdtdE_0
+
+def dYdt_0_func_gauss(t, E, N, mu_t, sig_t):
+    dYdt_0 = N/np.sqrt(2*np.pi)/sig_t * np.exp(-0.5 * ((t-mu_t)/sig_t)**2)
+    return dYdt_0
+
+'''
```

Comment:
Docstrings should go below the function declaration (calling ballabio_spectrum.__doc__ as you've written the code write now won't return anything, rather than returning the docstring as we would like).